Sources/Java2SwiftLib/StringExtras.swift

```diff
@@ -39,4 +39,18 @@ extension String {
   public func replacing(_ character: Character, with replacement: Character) -> String {
     return replacingOccurrences(of: String(character), with: String(replacement))
   }
+
+  public func optionalWrappedType() -> String? {
+    if self == "JavaOptionalInt!" {
+      return "Int32?"
+    } else if self == "JavaOptionalDouble!" {
+      return "Double?"
+    } else if self == "JavaOptionalLong!" {
+      return "Int64?"
+    } else if hasPrefix("JavaOptional<") {
+      return "\(String(self[index(startIndex, offsetBy: 13)..<lastIndex(of: ">")!]))?"
```

Comment:
the "13" is a bit magic number, can we .count on the string we're skipping instead please?